client/views/HistoryView.vue

```diff
@@ -66,8 +73,10 @@
       }
     },
 
-    created() {
-      this.load();
+    mounted() {
```

Comment:
Yes, because watcher will only trigger loading on refresh i.e. direct landing on history, but if we go then to another route and then come back to history we need 'mounted' to load data because watcher won't since app had already been initialized.